<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>847</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 236);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1031</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="search">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/images.png</normaloff>../../Downloads/images.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:(file:///home/ehab/Desktop/download.png);
</string>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/download (1).jpeg</normaloff>../../Downloads/download (1).jpeg</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Profile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>signOut</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="notification">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>20</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:(file:///home/ehab/Desktop/download.jpeg);
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/appointment-reminders.png</normaloff>../../Downloads/appointment-reminders.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QToolButton" name="Add2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/images (1).jpeg</normaloff>../../Downloads/images (1).jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>491</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);
</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="sortN">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>sort by name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortA">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>sort by appointment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Add">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>630</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(32, 74, 135);</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QListWidget" name="task_view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>471</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>14</pointsize>
       <weight>9</weight>
       <italic>true</italic>
       <bold>false</bold>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 75 italic 14pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>-7</number>
     </property>
     <property name="text">
      <string>Tasks</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="All">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>451</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(138, 226, 52);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>-7</number>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="done">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>431</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
